docker-compose.override.yml

```diff
@@ -0,0 +1,5 @@
+version: '3.7'
+services:
+  app:
+    volumes:
+      - .:/code:ro
```

Comment:
Azért módosítottam korábban nem readonly-ra, mert nálam a teljes setup docker-ben megy VSCode-dal, nem telepítettem külön az összes package-et a gépemre. Ezért szerintem ez jobb lenne úgy, hogy staging/production readonly, a fejlesztői változat pedig írható is.

Reply:
Jogos

Reply:
A prod-ban nem is kell mountolni, mert a prodnal hasznalt Dockerfile COPYzza a forraskodot.